pkg/traces/promsdprocessor/factory.go

```diff
@@ -22,13 +22,20 @@ const (
 	OperationTypeUpdate = "update"
 	// OperationTypeUpsert does both of above
 	OperationTypeUpsert = "upsert"
+
+	PodAssociationIPLabel       = "ip"
+	PodAssociationOTelIPLabel   = "net.host.ip"
+	PodAssociationk8sIPLabel    = "k8s.pod.ip"
```

Comment:
We should write up some docs with recommendations around this, e.g. inject `POD_IP` with the downward API, then set  `OTEL_RESOURCE_ATTRIBUTES=k8s.pod.ip=$(POD_IP)`

Reply:
Good idea. I'll probably add it to the config docs. We plan on merging agent config from here and from [tempo](https://grafana.com/docs/tempo/latest/grafana-agent/), but I think the config reference will do for now.